src/main/java/edu/iis/powp/events/predefine/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +16,9 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentPlotter());
+		if(e.getActionCommand() == "Figure Joe 1" )

```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.